services/swagger/swagger.service.js

```diff
@@ -61,15 +61,24 @@ module.exports = class SwaggerValidatorService extends BaseJsonService {
       },
     })
   }
-
   async handle({ scheme, url }) {
     const json = await this.fetch({ scheme, urlF: url })
     const valMessages = json.schemaValidationMessages
 
-    if (!valMessages || valMessages.length === 0) {
```

Comment:
Thanks for this! While you're updating this service, would you mind also extracting this logic (and the helper function) into a `transform` function that returns valid/invalid, and also move the color determination to the `render` function?

Reply:
Will do, for clarification: the helper method should also be inside the transform or can it stay outside?

Reply:
I think it can probably all go inside a single transform function